CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# ChangeLog for HDMI CEC Driver Library

```

Comment:
change-log should not change generated during a PR.. That's just a merge of fixes to develop..

A release is when a release engineer actually performs a release with a tag, that then causes the changelog to be generated.

1.1.0 is not been released yet on a release branch.

See the flow prcess -> https://github.com/rdkcentral/ut-core/discussions/43



This uring release process, not during a PR

Reply:
if this is the first commit, it shouldn't exist.

Reply:
Removed this file.